trading_cost.py

```diff
@@ -0,0 +1,64 @@
+"""
+A file to build cost objects.
+"""
+
+class CurrencyCost:
+    """
+    Save information in a currency trading.
+    """
+    def __init__(self, basic_ins, instruments):
+        self.basic_ins = basic_ins
+        self.cost = 0
+        self.profit = 0
+        self.quantity = {
+            instrument : 0 for instrument in instruments}
+        self.value = {
+            instrument : 0 for instrument in instruments}
+        self.value[basic_ins] = 1
+        self.quantity[basic_ins] = 1
```

Comment:
This is initial settings before tradings, means all asset are in basic_account and are not traded.